.github/workflows/test.yml

```diff
@@ -59,9 +59,9 @@ jobs:
       run:
         working-directory: openemr-devops/docker/openemr
     env:
-      COMPOSE_PROFILES: "${{ startsWith(matrix.docker-dir, 'flex-') && (matrix.docker-dir == 'flex-edge' && 'kcov' || 'flex') || 'prod' }}"
+      COMPOSE_PROFILES: "${{ startsWith(matrix.docker-dir, 'flex-') && 'flex' || 'prod' }}"
```

Comment:
note purposefully did this for now (yanked out the coverage PR for now and will put it back in after get other stuff working)